src/src/Environment/Docker.php

```diff
@@ -24,12 +24,12 @@ public function __construct( OutputInterface $output, Cache $cache ) {
 	public function find_docker(): string {
 		$docker = 'docker';
 
-		$docker_version = trim( shell_exec( $docker . ' --version' ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.system_calls_shell_exec
+		$docker_version = trim( shell_exec( $docker . ' --version 2>/dev/null' ) ?? '' ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.system_calls_shell_exec
 
 		if ( $docker_version ) {
 			return $docker;
 		} else {
-			throw new \RuntimeException( 'Could not find docker' );
+			throw new \RuntimeException( 'It seems Docker is not installed. Docker is needed to run some test types.' );
```

Comment:
Minor nit but I think we can direct them to the Docker docs here (doesn't need to be an explicit URL) but something like:

```suggestion
			throw new \RuntimeException( 'It seems Docker is not installed. Docker is needed to run some test types. For more information, please see the official documentation on installing the Docker engine for your operating system.' );
```